ressources/modeles/cocktail_modale.html

```diff
@@ -5,7 +5,7 @@
 
     <section id="recette">
         <div id="presentation">
-            <img id="illustration" src="https://picsum.photos/500/300" alt="Illustration du cocktail sélectionné">
+            <img id="illustration" src="" alt="Illustration du cocktail sélectionné">
```

Comment:
Pour que le linter passe:
```suggestion
            <img id="illustration" alt="Illustration du cocktail sélectionné">
```